test/test_cromwell_workflow.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
```

Comment:
`set -e` on this script and the following one

Reply:
Done.